components/network/quagga/Makefile

```diff
@@ -0,0 +1,86 @@
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+
+#
+# Copyright (c) 2006, 2019, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2020, Olaf Bohlen <olbohlen@eenfach.de>
+#
+BUILD_BITS= 32
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		quagga
+COMPONENT_VERSION=	0.99.19
+COMPONENT_REVISION=     1
+COMPONENT_SRC=          $(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=      $(COMPONENT_SRC).tar.gz
+IPS_COMPONENT_VERSION=	$(COMPONENT_VERSION)
+COMPONENT_PROJECT_URL=	http://www.quagga.net/
+COMPONENT_ARCHIVE_HASH=	\
+    sha256:bc5052d285af6e7aac42984741e74261637c57c4a02032bf05de7a5ea3a1afe0
+COMPONENT_ARCHIVE_URL=	https://download.savannah.gnu.org/releases/quagga/$(COMPONENT_ARCHIVE)
+COMPONENT_SIG_URL=	$(COMPONENT_ARCHIVE_URL).asc
+COMPONENT_BUGDB=	service/quagga
```

Comment:
COMPONENT_BUGDB, COMPONENT_ANITYA_ID, IPS_COMPONENT_VERSION are not necessary.

Reply:
fixed